frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts

```diff
@@ -219,21 +223,21 @@ export class CastDragAndDrop implements OnInit {
 
     // Sort positions by position order
     this.castPositions = this.castPositions.sort(
-        (a, b) => a.pos.order < b.pos.order ? -1 : 1 );
+        (a, b) => a.pos.order < b.pos.order ? -1 : 1);
     // Sort positions inside cast
     this.cast.filled_positions = this.cast.filled_positions
-      .filter(filledPos => this.castPositions.find(
-          castPos => castPos.pos.uuid === filledPos.position_uuid))
-      .sort((a, b) => {
-        const castPositionOrderA = this.castPositions.find(
-            castPos => castPos.pos.uuid === a.position_uuid).pos.order;
-        const castPositionOrderB = this.castPositions.find(
-            castPos => castPos.pos.uuid === b.position_uuid).pos.order;
-        return castPositionOrderA < castPositionOrderB ? -1 : 1;
-      });
+        .filter(filledPos => this.castPositions.find(
+            castPos => castPos.pos.uuid === filledPos.position_uuid))
+        .sort((a, b) => {
+          const castPositionOrderA = this.castPositions.find(
+              castPos => castPos.pos.uuid === a.position_uuid).pos.order;
+          const castPositionOrderB = this.castPositions.find(
+              castPos => castPos.pos.uuid === b.position_uuid).pos.order;
+          return castPositionOrderA < castPositionOrderB ? -1 : 1;
+        });
 
     for (let posIndex = 0; posIndex < this.cast.filled_positions.length;
-        posIndex++) {
+         posIndex++) {
       const filledPos = this.cast.filled_positions[posIndex];
       const uiPos = this.castPositions[posIndex];
       let maxDancerIndex = 0;
```

Comment:
I guess it is purposeful to line up with the prior line's parameters?

Reply:
same